src/js/tech/ChromecastTech.js

```diff
@@ -364,6 +364,33 @@ ChromecastTech = {
       return undefined;
    },
 
+   /**
+    * This function is "required" when implementing {@link external:Tech} and is supposed
+    * to return a mock
+    * {@link https://developer.mozilla.org/en-US/docs/Web/API/TimeRanges|TimeRanges}
+    * object that represents the portions of the current media item that has playable
+    * content. However, the Chromecast API does not currently provide a way to determine
+    * how much the media item has playable content, so we'll just assume the entire video
+    * is an available seek target.
+    *
+    * The risk here lies with live streaming, where there may exist a sliding window of
+    * playable content and seeking is only possible within the last X number of minutes,
+    * rather than for the entire video.
+    *
+    * Unfortunately we have no way of detecting when this is the case. Returning anything
+    * other than the full range of the video means that we lose the ability to seek during
+    * VOD.
+    *
+    * @returns {TimeRanges} always returns a `TimeRnages` object with one `TimeRnage` that
```

Comment:
A couple of typos here: `TimeRnages` and `TimeRnage`

Reply:
Damn, I make a lot of those 😮